manageiq-operator/deploy/operator.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: manageiq-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: manageiq-operator
+  template:
+    metadata:
+      labels:
+        name: manageiq-operator
+    spec:
+      serviceAccountName: manageiq-operator
+      containers:
+        - name: manageiq-operator
+          # Replace this with the built image name
+          image: docker.io/chainzeeli/manageiq-operator:v0.0.1
```

Comment:
I think it's safe to just change this to `docker.io/manageiq/manageiq-operator` now.